superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -201,8 +201,8 @@ export function estimateQueryCost(queryEditor) {
   };
 }
 
-export function clearInactiveQueries() {
-  return { type: CLEAR_INACTIVE_QUERIES };
+export function clearInactiveQueries(interval) {
```

Comment:
Where is this function being called with an `interval`?

Reply:
Good catch. I pushed the missing part.